Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -38,6 +38,7 @@ protected function configure()
             ->setName('doctrine:fixtures:load')
             ->setDescription('Load data fixtures to your database')
             ->addOption('append', null, InputOption::VALUE_NONE, 'Append the data fixtures instead of deleting all data from the database first.')
+            ->addOption('set', null, InputOption::VALUE_OPTIONAL, 'Only fixtures with this set attribute to the doctrine.fixtures.loader tag will be loaded')
```

Comment:
Should we allow this option to be an array of sets? Or may be also allow repeated use of the set option to use more than one set?

Reply:
hm maybe, i can add this if you want.
I can't really think of a use case where you would want this.
I think the main goal was to load fixtures per environment i don't know why you would wanna load multiple tags, just create a new tag then :)
for me it's not necessary.
But let me know if i need to add it